### ISSUE TITLE
Some pathway , eg carbon metabolism ko01200 does not return with fold change

### ISSUE THREAD

**Issue by liuxianghui** [datapplab/pathview#19]
Dear Luo:
I am surprised to find that some pathway, eg carbon metabolism ko01200, can not be plotted with fold change data.  It just give a n empty default pathway.
However, other pathways are ok.
Please kindly suggest, 
the following is the command and output:
pathview(gene.data  = names(geneList2), gene.idtype="kegg", pathway.id = "pfl01200",species= "pfl",limit = list(gene=c(min(geneList2),max(geneList2)), cpd=1))
Note: Mapping via KEGG gene ID (not Entrez) is supported for this species,
it looks like "PFL_4445"!
Info: Working in directory /Users/liuxianghui/work/biofilm3species
Info: Writing image file pfl01200.pathview.png
Info: some node width is different from others, and hence adjusted!

**Comment by Elseedy28** [comment]
Hi i am facing the same problem but i have the same problem for all pathway 
i have aquestion abiut the command you are using as i am using another one ( the one on the pathview's tutorial 
could you please tell me what is the meaning of species pfl?
thanks

**Comment by raw937** [comment]
Are you still having this issue?

This project is no longer under development here.
Please check the new fork here.
https://github.com/Kannapolis-Bioinformatics/pathview

If you have the same issues place in:
https://github.com/Kannapolis-Bioinformatics/pathview/issues

many thanks,
Rick

**Comment by Elseedy28** [comment]
Yes i still have the same issue but i will try what you recommend and i
will be back to you

Thanks
Heba
